Mathlib/MeasureTheory/Constructions/UnitInterval.lean

```diff
@@ -28,7 +30,73 @@ instance : IsProbabilityMeasure (volume : Measure I) where
     rw [Measure.Subtype.volume_univ nullMeasurableSet_Icc, Real.volume_Icc, sub_zero,
       ENNReal.ofReal_one]
 
+lemma measurableEmbedding_coe : MeasurableEmbedding ((↑) : I → ℝ) where
+  injective := Subtype.val_injective
+  measurable := measurable_subtype_coe
+  measurableSet_image' _ := measurableSet_Icc.subtype_image
+
+lemma volume_eq {s : Set I} : volume s = volume (Subtype.val '' s) :=
```

Comment:
Sorry, last change. I realized this is a more appropriate name, and it will need to be changed throughout.
```suggestion
lemma volume_apply {s : Set I} : volume s = volume (Subtype.val '' s) :=
```